client/mainmenu/CMainMenu.cpp

```diff
@@ -198,7 +199,7 @@ static std::function<void()> genCommand(CMenuScreen * menu, std::vector<std::str
 				case 2:
 					return std::bind(CMainMenu::openLobby, ESelectionScreen::loadGame, true, nullptr, ELoadMode::CAMPAIGN);
 				case 3:
-					return std::bind(CInfoWindow::showInfoDialog, CGI->generaltexth->translate("vcmi.mainMenu.tutorialNotImplemented"), std::vector<std::shared_ptr<CComponent>>(), PlayerColor(1));
+					return std::bind(CMainMenu::startTutorial);
```

Comment:
This should load tutorial, not start it

Reply:
honestly, I'm not sure that I understand how this should work. Does it load some hardcoded savegame?

Reply:
No. It shows only games made in tutorial.